qt/applications/workbench/workbench/widgets/settings/presenter.py

```diff
@@ -76,18 +75,18 @@ def action_section_changed(self, new_section_pos):
 
         self.current.show()
 
-    def action_save_settings_button(self):
-        if not self.ask_before_close or self.view.ask_before_close():
-            ConfigService.saveConfig(ConfigService.getUserFilename())
-            self.parent.config_updated()
-            self.view.close()
-
     def action_open_help_window(self):
         InterfaceManager().showHelpPage('qthelp://org.mantidproject/doc/workbench/settings.html')
 
-    def refresh_workspaces(self):
+    def view_closing(self):
         """
-        Refreshes the workspaces shown, so that if the invisible workspaces
-        setting was changed the effect will be reflected in the workspacewidget
+        Saes te mantid settings and updates updates te parent
```

Comment:
I think this is what it is supposed to mean.

```suggestion
        Saves the mantid settings and updates updates the parent
```